src/app/pages/OnDemandAudioPage/OnDemandAudioPage.jsx

```diff
@@ -49,20 +51,30 @@ const StyledGelWrapperGrid = styled(GelPageGrid)`
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
     padding-top: ${GEL_SPACING_TRPL};
   }
+  @media screen and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    display: flex;
+    flex-direction: row;
+  }
 `;
 
 const StyledGridItemParagraph = styled(Grid)`
-  @media (min-width: 22.5rem) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+  @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN} and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
     grid-template-columns: repeat(4, 1fr);
     grid-column-end: span 4;
   }
 `;
 
 const StyledGridItemImage = styled(Grid)`
-  @media (min-width: 22.5rem) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+  @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
     grid-template-columns: repeat(2, 1fr);
     grid-column-end: span 2;
   }
+  @media screen and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```

Comment:
what is the purpose of this media-query?

Reply:
This media query targets Internet Explorer browsers. For some reason, grid fallbacks are not applying here unless we explicitly target internet explorer